AZMCode/spotify-downloader
# Switch to rich (#1098)

Authored by @phcreery 

* initial changed for rich
* fixes
* comment consistency
* thx BetterCodeHub
* unnecessary comment
* descriptions/comments
* remove unnecesary try in main.py
* update pytest test cases
* Delete tmp
* clean up merge issues
* regenerate cassettes and verify pytest passes
* more descriptive error reporting
* update id3 function from `master`
* Adjust config
* Fixed regressions
* Updated step names
* fix testing
* cleanup merge conflicts
* fix more testing issues
* remove 1 mypy warning
* clean up SizedTextColumn
* fix typting
* fix merge conflicts
* coreected song naming
* fix cassetes
* suggested fixes and formatting
* flake8 compliant formatting
* remove residual unused function
* fix test to not call redundant function removed in 3ec3de4

Co-authored-by: Silverarmor <23619946+Silverarmor@users.noreply.github.com>
Co-authored-by: Andrzej Klajnert <github@aklajnert.pl>
Co-authored-by: Michael George <MikhailZex@gmail.com>
Co-authored-by: Jakub <42355410+xnetcat@users.noreply.github.com>
